lib/ansible/modules/packaging/os/mas.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python -tt
+
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Michael Heap
+# Written by Michael Heap <m@michaelheap.com>
+# Based on yum module written by Seth Vidal <skvidal at fedoraproject.org>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+module: mas
+author:
+    - "Michael Heap (@mheap)"
+version_added: "2.4"
+short_description: Manage applications via the Mac App Store
+description:
+    - Manage applications via the Mac App Store
+options:
+    id:
+        description:
+        - The ID of the package you want to install
+        - This can be found by running C(mas search APP_NAME) on your machine
+        required: true
+    state:
+        description:
+          - C(present) will make sure the package is installed.
+        required: false
+        choices: [ present ]
```

Comment:
How about 'absent' as state, which make this module complete.

Reply:
The underlying `mas` tool does not support removals. only installations

Reply:
Could you please add the note in documentation mentioning this limitation ?